api/init/graphqlapi/batch.py

```diff
@@ -0,0 +1,15 @@
+import docker
+
+
+def execute_batch(batch_id):
+    container_name = f'mobydq-batch-{batch_id}'
+    client = docker.from_env()
+    client.containers.run(
+        name=container_name,
+        image='mobydq-scripts',
+        network='mobydq-network',
+        links={'mobydq-graphql': 'mobydq-graphql'},
```

Comment:
@ThomasGassmann as discussed with @PascalHonegger I think we can remove the `links` argument here as `network` should do the job of connecting containers.

Reply:
Updated in 1238dd452bb7c3b3b3bd5173e722eafb8a33551c.